x-pack/plugins/watcher/public/models/index.d.ts

```diff
@@ -9,3 +9,20 @@ declare module 'plugins/watcher/models/watch' {
 declare module 'plugins/watcher/models/watch/threshold_watch' {
   export const ThresholdWatch: any;
 }
+declare module 'plugins/watcher/models/watch/json_watch' {
+  export const JsonWatch: any;
+}
+
+declare module 'plugins/watcher/models/execute_details/execute_details' {
+  export const ExecuteDetails: any;
+}
+
+declare module 'plugins/watcher/models/watch_history_item' {
+  export const WatchHistoryItem: any;
+}
+
+// TODO: Remove once typescript definitions are in EUI
+declare module '@elastic/eui' {
+  export const EuiCodeEditor: React.SFC<any>;
```

Comment:
wondering why these are needed.  I didn't need to add any eui types for my stuff (maybe I'm missing something)

Reply:
I think it's only for certain eui components.

see: https://github.com/elastic/kibana/blob/master/TYPESCRIPT.md#how-to-fix-common-typescript-errors

this may not be the correct place to put it. let me know what you think.